fern/docs.yml

```diff
@@ -6,64 +6,167 @@ instances:
         owner: fern-api
         repo: fern
         branch: main
-  # - url: fern-beta.docs.buildwithfern.com/learn
-  #   custom-domain: staging.buildwithfern.com/learn
 title: Fern | Documentation
 
-tabs:
-  overview:
-    display-name: Overview
-    icon: "globe"
-  sdks:
-    display-name: SDKs
-    slug: sdks
-    icon: code
-  docs:
-    display-name: Docs
-    slug: docs
-    icon: book
-
-navigation:
-  - tab: overview
-    layout:
+navigation: 
+  - section: Home
+    contents: 
       - page: Introduction
-        path: ./docs/pages/overview/introduction.mdx
-      - page: Why Fern?
-        path: ./docs/pages/overview/why-fern.mdx
-      - page: Get started
-        path: ./docs/pages/overview/get-started.mdx
-      - section: Define your API
+        path: ./overview/intro.mdx
+  - section: SDKs
+    contents: 
+      - page: Overview
+        path: ./overview/dummy.mdx
+      - page: Quickstart
+        path: ./overview/dummy.mdx
+      - section: Features
+        contents: 
+          - page: Strongly typed
+            path: ./overview/dummy.mdx
+          - page: Idiomatic method names
+            path: ./overview/dummy.mdx
+          - page: Schema Validation
+            path: ./overview/dummy.mdx
+          - page: Auto-Pagination
+            path: ./overview/dummy.mdx
+          - page: Retries with backoff
+            path: ./overview/dummy.mdx
+          - page: Webhook Signature Verification
+            path: ./overview/dummy.mdx
+          - page: Forward Compatibility  
+            path: ./overview/dummy.mdx
+          - page: Multipart Form Data
+            path: ./overview/dummy.mdx
+          - page: Idempotency Headers
+            path: ./overview/dummy.mdx
+          - page: Server-Sent Events
+            path: ./overview/dummy.mdx
+          - page: Integration Tests
+            path: ./overview/dummy.mdx
+          - page: Code Snippets
+            path: ./overview/dummy.mdx
+          - page: Augment with custom code
+            path: ./overview/dummy.mdx
+          - page: Merging multiple APIs
+            path: ./overview/dummy.mdx
+          - page: GitHub integration
+            path: ./overview/dummy.mdx
```

Comment:
In general, throughout all sections, I suggest picking either sentence case or title case or sticking with it for a consistent look and feel. Right now, casing seems random - "Merging multiple APIs", "Integration Tests", "GitHub integration", "Server-Sent Events", etc. This is also something to look out for in headings (h1, h2, etc).